app/assets/javascripts/channels/learning_trails.js

```diff
@@ -0,0 +1,16 @@
+$('input[type=checkbox]').change(function(){
+    counter = 0;
+    clicked = $(this).data('index');
+    $('input[type=checkbox]').each(function(){
+        if($(this)[0].checked){
+            counter++;
+        }
+    });
+    if(counter==3){
+        toDisable = clicked;
+        while(toDisable==clicked){
+            toDisable=Math.round(Math.random()*2);
+        }
+        $("input:eq("+toDisable+")")[0].checked = false;
+    }
+});
```

Comment:
What does this function do?

Reply:
This was added while testing some css switches, it is unused and will be removed.  I will check the diffs more carefully next time.